test/plugin/currencyPlugin/fixtures.json

```diff
@@ -5,7 +5,7 @@
     "WALLET_FORMAT": "bip32",
     "Test Currency code": "BTC",
     "key": [39, 190, 34, 129, 208, 32, 145, 88, 191, 217, 226, 98, 183, 16, 52, 150, 52, 53, 31, 137, 164, 40, 236, 146, 128, 107, 129, 59, 192, 240, 40, 238],
-    "xpub": "xpub69FqMgncSEcrs989ejBWTBBcDNFDqkwEd7y53pVeXm8368TNfb9jCd2ne3ccpx9vvgBdpv79Edc69i2Q69kXtrdmLcQM8seffnCXzwzvWa6",
+    "xpub": "xpub661MyMwAqRbcF6JxG5NqmWiCbURzYtg95A5T7m6bdJ27FHDuLcVHmAg4unEMvdNi5VniUWgxxDJM5odBjUUzuSNCciED3sbfdX37NsdKTiQ",
```

Comment:
You were right with your assumption, the default master path that is derived has changed.
So even though it seems problematic, it shouldn't effect anything since we never use this key directly anyway.